apidoc/Facebook.yml

```diff
@@ -1294,6 +1299,89 @@ properties:
     since: "6.0.0"
     platforms: [iphone, ipad, android]
 
+  - name: EVENT_NAME_COMPLETED_REGISTRATION
+    summary: Log this event when a user has completed registration with the app.
+    type: String
+    permission: read-only
+    since: "8.1.0"
```

Comment:
This should be specified per platform.

Reply:
@janvennemann  I couldn't understand, what you are saying. Can you explain please? Thanks!

Reply:
Sorry, totally missed this!

What is the version number referring to here? Module version or SDK version? I assumed it's the module version so it needs different versions specified for Android and iOS because there is no module version 8.1.0 for iOS.